### ISSUE TITLE
Iphone5 ios8.1 Clock icon blur issue

### ISSUE THREAD

**Issue by Tommy1001** [nickfrey/IconBundles#2]
Dear Nick:
         Recently I updated IconBundles to version 0.2-1 through Cydia BigBoss and noticed that my clock icon is now looks blurred instead of looking clear as before. My clock icon name is still as before "com.apple.mobiletimer@2x.png" placed in the same "IconBundles" directory under my theme folder.

If I recall correctly, the previous version of IconBundles did not have this clock icon blur display issue... Not sure if you have noticed or experienced similar situation with 0.2-1. If it is an issue, hopefully you have a solution for it sir : )

Much thanks and regards
Tommy

**Comment by nickfrey** [comment]
Yup, this was a regression in the latest version that I didn't catch. I'm working on a fix.

**Comment by Tommy1001** [comment]
Thank you so much sir Nick : ) Much appreciation! Just want to say it is all due to your wonderful creation "IconBundles" which have made our own custom theming possible! Always wanted to swap out the default ios clock icon. Thanks again for working hard for all of us sir!

Cheers
Tommy
